src/transpilation/bundle.ts

```diff
@@ -22,7 +24,9 @@ local function require(file, ...)
     end
     if ____modules[file] then
         local module = ____modules[file]
-        ____moduleCache[file] = { value = (select("#", ...) > 0) and module(...) or module(file) }
+        ____moduleCache[file] = { value = (${isLua50 ? "table.getn(arg)" : 'select("#", ...)'} > 0) and module(${
```

Comment:
This is unreadable, please pre-format this entire table literal beforehand

Reply:
Done with https://github.com/TypeScriptToLua/TypeScriptToLua/pull/1263/commits/f2563cc76ddd7a638a71e437448faec4669f1aed.